### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by ashishagarwal2023** [N3RDIUM/dotfiles#1]
Never used dotfiles before. I may try this, how?

**Comment by N3RDIUM** [comment]
I will add an install guide soon. Till then, you can tell me what your current desktop config is.

Generally, these steps are followed while installing dotfiles:
1. Backup your existing config
2. Install GNU Stow and other required programs
3. Clone this repo, `cd` into it and run `stow --adopt .`

Please be careful, doing a single mistake can result in irreversible damage to your system unless you back up your existing config. This has happened to me once!

**Comment by ashishagarwal2023** [comment]
> I will add an install guide soon. Till then, you can tell me what your current desktop config is.
> 
> Generally, these steps are followed while installing dotfiles:
> 
> 1. Backup your existing config
> 2. Install GNU Stow and other required programs
> 3. Clone this repo, `cd` into it and run `stow --adopt .`
> 
> Please be careful, doing a single mistake can result in irreversible damage to your system unless you back up your existing config. This has happened to me once! Mind you, this is my personal config, and it may take some tweaking to get it perfectly suited to your requirements.

yeah i have windows 11 and ubuntu iso in 2 pendrives + if ubuntu dies i have windows 11 dualboot :skull: dont worry XD
